js/sdk/src/utils/truncateManager.js

```diff
@@ -5,6 +5,17 @@ export const truncateAddress = (address) => {
   )}`
 }
 
-export const truncateString = (string) => {
-  return `${string.slice(0, 20)}...`
+export const truncateStringToLength = (string, index) => {
+  return `${string.substring(0, index)}...`
+}
+
+export const truncateForUi = (string, index, stringLength, maxLength) => {
```

Comment:
`index, stringLength, and maxLength` should have default values as they seem to be re-used with mostly the same values